orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/pipeline/DeleteScalingPolicyStage.groovy

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.kato.pipeline
```

Comment:
You're making Scaling policy's generic, no?

If generic, I would put them under `c.n.s.o.clouddriver.pipeline.scaling`, else `c.n.s.o.clouddriver.provider.aws`. There is already a `ModifyAwsScalingProcessStage` there.